package/yast2-apparmor.changes

```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Wed Jul 17 07:25:37 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Avoid crashes due to dead code related to notify settings, which
```

Comment:
Crashes? I though it was only a warning in the console.

Reply:
Yep, for me were only _warnings_ int the console. However, in the bug report is stated 

> Error (Can't locate Immunix/Notify.pm in @INC) shows up and the "Manage Existing Profiles" is broken.

That's why I used that word. Although it worked _perfectly_ in my tests.